CONTRIBUTING.md

```diff
@@ -42,11 +42,33 @@ vitess/base
 
 ## Breaking changes
 
-We use the [Kotlin binary compatibility validator][2] to check for API changes. If 
-a change contains an API change and breaks the build, run the `:apiDump` task and 
-commit the resulting changes to the `.api` files. `.api` files should not have 
-removals and additions in the same change so that downstream apps do not immediately
-run into backwards-compatibility issues.
+We integrate both [Kotlin Binary Compatibility Validator][2] and [Revapi][3] into our build 
+pipeline to automatically detect breaking changes that could affect existing clients.
+
+### Kotlin Binary Compatibility Validator
+This tool checks for changes to the public binary API. The plugin generates a dump of the
+current API and compares it to the previous version. If a binary incompatibility is detected, 
+the build will fail. If the change is intentional, you can run the `:apiDump` task and commit 
+the resulting changes to the `.api` files. `.api` files should not have removals and additions 
+in the same change so that downstream apps do not immediately run into backwards-compatibility 
+issues.
+
+This runs as part of `gradle check` task, or you can call directly with `gradle apiCheck`.
+
+### Revapi
```

Comment:
Could we link out to the repo/docs where this tool is from?

Reply:
I did in the beginning paragraph, it's marked as [3].